Indicators/RelativeStrengthIndex.cs

```diff
@@ -99,7 +99,13 @@ protected override decimal ComputeNextValue(IndicatorDataPoint input)
             }
 
             var rs = AverageGain / AverageLoss;
-            return 100m - (100m / (1 + rs));
+            if (rs == -1m)
+            {
+                // prevent division by zero
+                return 100m;
```

Comment:
Fix looks good 👍 
When the value of `rs` is getting close to -1, it starts returning values > 100 and later < -100. Curious if we should be limiting it to 0 to 100?

I get the following logs when adding a `Log.Trace($"{rsi.Current}");` in the unit test
```
200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 97.19626168224299065420560748
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 97.80219780219780219780219780
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 98.70129870129870129870129870
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 100
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 101.81818181818181818181818182
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 104.34782608695652173913043478
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 107.89473684210526315789473684
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 109.375
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 111.11111111111111111111111111
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 113.63636363636363636363636364
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 116.66666666666666666666666667
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 121.42857142857142857142857143
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 127.27272727272727272727272727
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 137.5
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 150
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 175.00000000000000000000000000
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 249.99999999999999999999999999
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 400.00000000000000000000000003
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 100
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - -200.00000000000000000000000003
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - -49.99999999999999999999999999
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 0
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 24.999999999999999999999999998
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 40.000000000000000000000000001
20200316 22:12:10.712 Trace:: 2020-03-17T01:12:10 - 50
```

Reply:
@Martin-Molinero I updated the indicator to handle the negative averages which were returning RSI values outside of the expected `[0..100]` range.